src/connectors/pagination/connectPagination.js

```diff
@@ -111,6 +121,9 @@ export default function connectPagination(renderFn, unmountFn) {
             currentRefinement: helper.getPage() || 0,
             nbHits: 0,
             nbPages: 0,
+            pages: pager.pages(),
```

Comment:
During `init`, we know the values as there has been no search yet, so we can hardcode them:
 - pages: `[]`
 - isFirstPage: `true`
 - isLastPage: `true`

Reply:
sure, just had them here for consistency